examples/hello_world/src/main.rs

```diff
@@ -49,9 +49,20 @@ impl eframe::App for MyApp {
             }
             ui.label(format!("Hello '{}', age {}", self.name, self.age));
 
-            ui.image(egui::include_image!(
-                "../../../crates/egui/assets/ferris.png"
-            ));
+            // ui.image(egui::include_image!(
```

Comment:
Is this change to the example intentional? It looks like test code.

Reply:
oops! thanks for catching that 😅